AlarmPlus/Views/AlarmTableViewController.swift

```diff
@@ -9,12 +9,99 @@
 import UIKit
 
 class AlarmTableViewController: UITableViewController {
-
+    var scheduledAlarmEvents : [Date] = []
```

Comment:
Swift Style: move the colon to the left
`var scheduledAlarmEvents: [Date] = []`

if you are using Swift 4.2, you can declare it like this:

`var scheduledAlarmEvents = [Date]()`

The parens at the end tells Swift to immediately instantiate an empty array that will contain Date types.